plugins/signalmonitor/signalhistorymodel.h

```diff
@@ -64,6 +65,7 @@ class SignalHistoryModel : public QAbstractTableModel
 
 public:
     enum ColumnId {
+        MonitoredColumn = 0,
```

Comment:
The extra column needs a lots of space for just the checkbox, can we merge this into the object name column maybe?

Reply:
Will do

Reply:
The very minimum width for this column has to be check indicator width + sort indicator width + spacing in between, giving approximatively 40pixels here.
Ig this comes unacceptable for you, then the only option is to remove such column :(